roles/keystone/templates/etc/keystone/policy.json

```diff
@@ -1,5 +1,9 @@
 {
+{% if policy_for_horizon|default('False')|bool %}
+    "admin_required": "role:admin or is_admin:1 or role:cloud_admin",
```

Comment:
Cloud_admin should still be restricted to certain panels using the horizon-customization module. I am only seeing the panels that we allow in that module when logged in as cloud_admin. This policy_for_horizon fact should only be set when the customization module is enabled.